extension/src/popup/views/SignTransaction/Preview/Summary/index.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+
+import "./styles.scss";
+import { Icon } from "@stellar/design-system";
```

Comment:
same issue as above here

Reply:
fixed in efda2d157b9925c274d733854f10634fea9e495d